pkg/cluster/builder.go

```diff
@@ -154,6 +154,28 @@ func (b FileSpecBuilder) getBundles(manifestReader *manifests.Reader) (*releasev
 	return bundles.Read(b.reader, bundlesURL)
 }
 
+func (b FileSpecBuilder) getEksaRelease(mReader *manifests.Reader) (*releasev1.EksARelease, error) {
+	if b.bundlesManifestURL == "" {
+		// this shouldn't return an error at this point due to getBundles performing similar operations prior to this call
+		release, err := mReader.ReadReleaseForVersion(b.cliVersion.GitVersion)
+		if err != nil {
+			return nil, err
+		}
+		return release, nil
+	}
+
+	return &releasev1.EksARelease{
+		Date:              "2023-07-26 20:02:17.931339039 +0000 UTC",
+		Version:           b.cliVersion.GitVersion,
+		Number:            1,
+		GitCommit:         "",
+		GitTag:            "",
+		BundleManifestUrl: "",
+		EksABinary:        releasev1.BinaryBundle{},
+		EksACLI:           releasev1.PlatformBundle{},
```

Comment:
Also, why we these values just empty? Do they not matter? If so, shouldn't the function just return the version?